tests/integration/x/vm/test_genesis.go

```diff
@@ -119,7 +119,7 @@ func (s *GenesisTestSuite) TestInitGenesis() {
 			ctx = s.network.GetContext()
 			vmdb = statedb.New(
 				ctx, s.network.App.GetEVMKeeper(),
-				statedb.NewEmptyTxConfig(common.BytesToHash(ctx.HeaderHash())))
+				statedb.NewEmptyTxConfig())
```

Comment:
Are these changes needed in this PR? could we separate this into an isolated PR?